### ISSUE TITLE
Updating 4.0.1->4.3.1 creates new eslint errors

### ISSUE THREAD

**Issue by chemicalkosek** [wesbos/eslint-config-wesbos#144]
Steps to reproduce:

Clone repository: https://github.com/chemicalkosek/eslint-wesbos-4.3.1
It's just basic app from `create-next-app` with `eslint-config-wesbos@4.0.1`.
It also has a function with nullish coalescing being used.

`npm install`

Now run
`npm run lint`

Ignore the current eslint errors

Install 4.3.1 version of eslint-config-wesbos
`npm install eslint-config-wesbos@4.3.1`

Run eslint again:
`npm run lint`

Now see a new error pointing to the place where nullish coalescing is being used:
```bash
  21:52  error  Parsing error: Unexpected token ?
  ```

**Comment by wesbos** [comment]
Try update to the latest ESlint 8  (not 9 yet) - I removed the Babel parser so you are probably using an older version of ESlint that can't parse nullish coalescing

**Comment by chemicalkosek** [comment]
It has the latest eslint 8 version, Wes.

```bash
npm list | grep eslint

├── eslint-config-next@15.0.1
├── eslint-config-wesbos@4.3.1
├── eslint@8.57.1
```

**Comment by wesbos** [comment]
You are right. I should be specifying the parser version/

You can fix right away with this config:

```json
{
  "extends": "wesbos",
  "parserOptions": {
    "ecmaVersion": "2021"
  }
}

```

but Ill also push a change

**Comment by wesbos** [comment]
fixed in 4.3.2
